### PR TITLE
docs: update CONTRIBUTING.md with clearer instructions for install & changesets

### PR DESCRIPTION
**Pull request by dbarnett**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -79,7 +81,7 @@ pnpm astro:dev
 
 ## Some Guidelines to follow
 
-Make sure you have fulfilled the requirements and local development steps mentioned [here](../README.md#local-development-🧑🏻‍💻).
```
Comment (dbarnett) [inline docs/CONTRIBUTING.md:82]: This link was broken. Was it meant to link to the Local Development section in this same doc, or something else?

**Comment by dbarnett** [comment]
Just a variety of suggested edits for things that confused me while getting familiar with the project. Feel free to ignore or rework whatever parts of these changes to your personal tastes. =)

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (kareemmahlees) [reply docs/CONTRIBUTING.md:82]: Yes exactly.

**Review by kareemmahlees — APPROVED** [review]

**Comment by kareemmahlees** [comment]
@all-contributors please add @dbarnett  for doc

**Comment by kareemmahlees** [comment]
@dbarnett Hey, thank you for this informative change.

Although I have noticed some weird behavior with the `cargo install --path apps/core/src-tauri` command, It installs the dependencies and all is good but it also builds the application at `cargo/.bin` ( which is a broken build because `tauri` must build it to work ).

But I just remembered that the `pnpm tauri:dev` script automatically compiles the necessary dependencies without producing without installing the app binary at `cargo/.bin`.

I have fixed it [here](https://github.com/kareemmahlees/tablex/commit/f0fc949e5037782dcf63eba2b640c82793fb2fcd).

**Comment by dbarnett** [comment]
Excellent, thanks! IDK if the other command I ran did anything useful, just didn't error out. I can try cleaning out the other cargo files and see if it affects anything.

**Pull request merged**